data/160/0105/en_US.json

```diff
@@ -1,13 +1,13 @@
-{
-	"160": {
-		"0105": {
-			"name": "Unknown",
-			"message": "",
-			"short_description": "Unknown cause.",
-			"long_description": "Unknown cause.",
-			"short_solution": "Unknown solution.",
-			"long_solution": "Unknown solution.",
-			"support_link": "https://preten.do/160-0105"
-		}
-	}
+{
+	"160": {
+		"0105": {
+			"name": "Unknown",
+			"message": "There is a problem with the USB storage device.\n\nPlease contact the manufacturer of the device.\nCheck the device documentation for contact details.",
+			"short_description": "USB device error",

```

Comment:
If a short description is being added, then a long one should also be added. The `short` fields are designed for use by the Discord bot. The `long` fields are used for the website. Only having one of them makes the data unavailable by the other.

You can see a good example of how to format these fields here https://github.com/PretendoNetwork/error-codes/blob/master/data/104/2210/en_US.json

This goes for all cases of this, as there are multiple in this PR

Reply:
Should be fixed up now
Speaking of 104-2210, I've completed that one too. Also, for some reason it doesn't appear on Bandwidth when asking in the Discord server. Hasn't for a while. Just worth bringing it up now

Reply:
Bandwidth won't show any new error codes until a new release of this repo is made and Bandwidth is updated to use it

Reply:
Thanks for reminding me, the existing one must not have that error documented.